ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/encoding/BlockAccessListDecoder.java

```diff
@@ -72,7 +73,7 @@ public static BlockAccessList decode(final RLPInput in) {
               bcIn -> {
                 bcIn.enterList();
                 int txIndex = bcIn.readIntScalar();
-                Bytes postBalance = bcIn.readBytes();
+                Wei postBalance = Wei.fromHexString(bcIn.readBytes().toHexString());
```

Comment:
Is this one working ? I will prefer to avoid hex string 

```java
Wei.of(UInt256.fromBytes(bcIn.readBytes()))
```

Reply:
I also wasn't sure about using hex strings here due to performance implications, but postponed a proper solution. This one seems to work!